Caddyfile

```diff
@@ -0,0 +1,8 @@
+https://local.digitalcash.dev {
```

Comment:
why not just use `https://localhost`?

Reply:
Just a general best practice to use a real domain.

`localhost` is treated very special by browser security permissions and hence all the web meta-frameworks have a bunch of weird hacks to try to make it work.

As long as you don't do anything involving Cookies, "HTML5" APIs, etc it shouldn't matter in this particular instance. I don't care that much.